db/schema.rb

```diff
@@ -94,12 +94,13 @@
     t.uuid "parent_profile_id"
     t.boolean "external", default: false, null: false
     t.uuid "policy_id"
+    t.string "os_version"
     t.index ["account_id"], name: "index_profiles_on_account_id"
     t.index ["external"], name: "index_profiles_on_external"
     t.index ["name"], name: "index_profiles_on_name"
     t.index ["parent_profile_id"], name: "index_profiles_on_parent_profile_id"
     t.index ["policy_id"], name: "index_profiles_on_policy_id"
-    t.index ["ref_id", "account_id", "benchmark_id", "external", "policy_id"], name: "uniqueness", unique: true
+    t.index ["ref_id", "account_id", "benchmark_id", "external", "os_version", "policy_id"], name: "uniqueness", unique: true
```

Comment:
I guess we can drop `external` from the uniqueness, as the support for external policies has been dropped. `external` is can be only considered as a flag for initial policy profile.

Reply:
I'm fine with including this change with this PR.